src/text-fragment-utils.js

```diff
@@ -822,69 +822,38 @@ if (typeof goog !== 'undefined') {
 }
 
 /**
- * Extract color and background-color from ::target-text in the document.
+ * Replaces all occurence of the pseudo element ::target-text to a css class
+ * chrome-target-text
  *
- * @return {{backgroundColor: string, color: string}} - color and
- *     background-color from ::target-text
+ * @return {bool} - true iff we can create a css class from
```

Comment:
Oh the idea was to set the default style in the css class if `applyTargetTextStyle` returns false.

Reply:
But, I think we can remove the boolean and in a separate function just add the class with the default rules if the class is not there already.

Reply:
I don't think it will be that simple--it's possible that the publisher's rules won't cover the `mark`s we add; e.g., if the publisher has rules for `p::target-text` but then we add a `mark` that's not inside a `p`.

I think the right approach is probably just for us (or, rather, for polyfill users) to add a `.text-fragments-polyfill-target-text { ... }` entry--I think CSS behavior should make that only apply if the rules that this function generates don't cover it. (It might be dependent on the order they're declared in, though.)

Reply:
Yes sorry that's what I meant. The function will check if the publisher sets `.text-fragments-polyfill-target-text { ... }` with both `background-color` and `color` and if not, we will add the class with the missing attributes.

Reply:
Hmmm, ok. Feel free to handle this in a separate patch so we don't get blocked on this detail right now

Reply:
Ok sounds good